ui/components/app/signature-request/signature-request-header/signature-request-header.component.js

```diff
@@ -24,9 +46,33 @@ export default class SignatureRequestHeader extends PureComponent {
           ) : null}
         </div>
         <div className="signature-request-header--network">
-          <NetworkDisplay />
+          <PickerNetwork
+            as="div"
+            src={currentNetwork?.rpcPrefs?.imageUrl}
+            label={
+              providerConfig?.type === NETWORK_TYPES.RPC
+                ? providerConfig?.nickname ?? t('privateNetwork')
+                : t(getNetworkLabelKey(providerConfig?.type))
+            }
+            backgroundColor={BackgroundColor.transparent}
+            borderColor={BorderColor.borderMuted}
+            iconProps={{ display: Display.None }}
```

Comment:
@MaxwellDG @georgewrmarshall Do we have icons for our built in networks? I don't know if we want to introduce the letters here? Might cause user confusion.

Reply:
I don't believe there's icons for the built-in ones. The previous components just used small coloured circles. And I couldn't see any Sepolia or Goerli related images in Storybook. Maybe you know of some elsewhere you'd like me to use?

Reply:
I believe the icons are retrieved here 
```
src={currentNetwork?.rpcPrefs?.imageUrl}
```

Sepolia and Goreli test networks use a combination of html elements and color to create the icon but this adds complexity to how we display networks. I'll create a ticket to add the images and migrate them over. This should reduce the complexity if all networks are displayed using images.

Reply:
Created an issue here https://github.com/MetaMask/metamask-extension/issues/21142